apps/wallet/src/ui/app/pages/welcome/index.tsx

```diff
@@ -24,11 +24,7 @@ const WelcomePage = () => {
                     <div className="pt-6 rounded-20 bg-alice-blue shadow-wallet-content flex flex-col flex-nowrap items-center justify-center w-popup-width h-popup-height">
                         <BottomMenuLayout>
                             <Content className="flex flex-col flex-nowrap items-center p-7.5 pb-0">
-                                <Logo
-                                    size="normal"
-                                    className="text-hero mt-7.5"
-                                    txt={true}
-                                />
+                                <Logo className="text-hero mt-7.5" />
```

Comment:
If we're re-doing this can we also just make it stop taking `className` as a prop?